launch/tier4_simulator_launch/launch/simulator.launch.xml

```diff
@@ -25,6 +26,47 @@
     </include>
   </group>
 
+  <!-- perception module -->
+  <group>
+    <push-ros-namespace namespace="perception"/>
+    <!-- object recognition -->
+    <group if="$(var perception/enable_object_recognition)">
+      <push-ros-namespace namespace="object_recognition"/>
+      <!-- detection module -->
+      <!-- tracking module -->
+      <group>
+        <push-ros-namespace namespace="tracking"/>
+        <include file="$(find-pkg-share tier4_perception_launch)/launch/object_recognition/tracking/tracking.launch.xml">
+        </include>
+      </group>
+      <!-- prediction module -->
+      <group>
+        <push-ros-namespace namespace="prediction"/>
+        <include file="$(find-pkg-share tier4_perception_launch)/launch/object_recognition/prediction/prediction.launch.xml">
+          <arg name="use_vector_map" value="true" />
+        </include>
+      </group>
+    </group>
+
+    <!-- object segmentation module -->
+    <group>
+      <push-ros-namespace namespace="occupancy_grid_map"/>
+      <!-- Occupancy Grid -->
+      <include file="$(find-pkg-share laserscan_to_occupancy_grid_map)/launch/laserscan_to_occupancy_grid_map.launch.py">
+        <arg name="input_obstacle_pointcloud" value="true" />
+        <arg name="input_obstacle_and_raw_pointcloud" value="false" />
+        <arg name="input/obstacle_pointcloud" value="/perception/obstacle_segmentation/pointcloud" />
+        <arg name="output" value="/perception/occupancy_grid_map/map" />
+      </include>
+    </group>
+
+    <!-- traffic light module -->
+    <group if="$(var perception/enable_traffic_light)">
+      <push-ros-namespace namespace="traffic_light_recognition"/>
+      <include file="$(find-pkg-share tier4_perception_launch)/launch/traffic_light_recognition/traffic_light.launch.xml"></include>
+    </group>
+  </group>
+
```

Comment:
The only things that are simulated are detection and obstacle segmentation. Other than that, tracking and prediction are working as usual (no simulation), so should it is put them in simulation.launch?

Reply:
@yukkysaito 
In planning_simulator.launch.xml, perception launch is not called (planning/control launches are called).
Currently, we call perception launch (tracking, prediction, obstacle segmentation) from dummy_perception_publisher when using simple psim.

Now we want to use this perception launch with scenario sim as well, which does not depend on dummy_perception_publisher, so I moved them to simulator.launch.xml.
I'm ok to move to planning_simulator.launch.xml if you want

Reply:
>In planning_simulator.launch.xml, perception launch is not called (planning/control launches are called).
Currently, we call perception launch (tracking, prediction, obstacle segmentation) from dummy_perception_publisher when using simple psim.

Yes. I know :+1: 

What I mean is that tracking and prediction are not simulated. What is being simulated is the point cloud, the vehicle model, and detection.
There is no problem that point cloud, the vehicle model, and detection. are in simulation.launch. 
But there are things in simulation.launch that are not being simulated.
I'm not sure what is best. :thinking: